go.mod

```diff
@@ -9,7 +9,7 @@ require (
 	github.com/Azure/azure-sdk-for-go/sdk/azidentity v1.3.1
 	github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/authorization/armauthorization/v2 v2.1.1
 	github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/compute/armcompute/v5 v5.2.0
-	github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/containerservice/armcontainerservice/v4 v4.3.0
+	github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/containerservice/armcontainerservice/v4 v4.4.0-beta.1
```

Comment:
/hold

I don't know that we want to bump to beta releases of the SDK. What are the support guarantees around beta releases? Are they subject to breaking changes?

related to #2625

Reply:
I'm not sure we want to do this either; I created this PR partially to raise the question and partially to see if there are any technical issues for CAPZ using the beta release.